Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/adapter/proxy/AdapterPatientDiscoveryProxyWebServiceHelper.java

```diff
@@ -73,12 +74,20 @@ public PRPAIN201306UV02 formatGatewayResponsePRPAIN201305UV02(PRPAIN201305UV02 b
                     request.setPRPAIN201305UV02(body);
                     request.setNhinTargetCommunities(null);
 
-                    ServicePortDescriptor<AdapterPatientDiscoverySecuredPortType> portDescriptor = new AdapterPatientDiscoverySecuredServicePortDescriptor();
-                    CONNECTClient<AdapterPatientDiscoverySecuredPortType> client = CONNECTClientFactory.getInstance()
-                        .getCONNECTClientSecured(portDescriptor, url, assertion);
-
-                    response = (PRPAIN201306UV02) client.invokePort(AdapterPatientDiscoverySecuredPortType.class,
+                    ServicePortDescriptor portDescriptor;
+                    CONNECTClient client;
+                    if (NhincConstants.ADAPTER_PATIENT_DISCOVERY_SECURED_SERVICE_NAME.equals(sServiceName)) {
```

Comment:
Suggestion, pull out the added logic into a new method:  buildPortDescriptor(String sServiceName), that will pull out the added if/else statement.  Also rename this helper method.  Should be something like sendToPRPAIN201306UV02Adapter(...).  The current name "formatGatewayResponsePRPAIN201305UV02" isn't really what it does.  There is no formatting.